### ISSUE TITLE
Continually getting "Error: spawn shellcheck ENOENT"

### ISSUE THREAD

**Issue by leefr** [AtomLinter/linter-shellcheck#85]
Since last atom update, when editing any shell script continually get the error:

```
Error: spawn shellcheck ENOENT
    at exports._errnoException (util.js:890:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:182:32)
    at onErrorNT (internal/child_process.js:348:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

Currently running:
atom 1.11.2
linter-shellcheck 1.4.1

**Comment by Arcanemagus** [comment]
Do you have `shellcheck` installed on your system?

**Comment by leefr** [comment]
Not as a separate thing.  The only references I find on my system are for linter-shellcheck under my .atom folder.  This all seemed to be working fine for me in earlier versions and I had not edited any shell scripts in a couple weeks, but remember applying an atom update and updating a bunch of plugins in the interim, then hitting this issue yesterday.

**Comment by leefr** [comment]
Apparently I had it installed previously as a dep for a vim plugin or something that I removed and had shellcheck go away too.  Sorry for the false alarm!
